static/js/util/currency.js

```diff
@@ -15,3 +16,13 @@ const codesToOptions = R.compose(
 );
 
 export const currencyOptions = codesToOptions(cc.codes());
+
+const codeToCountryName = code => iso3166.country(code).name || '';
+
+const countryNameToCurrency = name => (
+  name === '' ? '' : cc.country(name)[0].code
```

Comment:
no, we're not defaulting to USD. The default is whatever the default currency for your country it.